mdbenchmark/mdengines/gromacs.py

```diff
@@ -35,10 +37,23 @@ def prepare_benchmark(name, relative_path, *args, **kwargs):
 
     filepath = os.path.join(relative_path, full_filename)
 
-    copyfile(filepath, benchmark[full_filename].relpath)
+    if kwargs["multidir"] == 1:
+        copyfile(filepath, benchmark[full_filename].relpath)
+    else:
+        for i in range(kwargs["multidir"]):
+            subdir = benchmark[MULTIDIRS[i]+"/"+full_filename].make()
```

Comment:
I assume our linters will complain about the style here. Instead it probably should read:

```suggestion
            subdir = benchmark[LOWERCASE_LETTERS[i] + "/" + full_filename].make()
```